### ISSUE TITLE
[Rails 6-1]

### ISSUE THREAD

**Issue by luisramos0** [openfoodfoundation/openfoodnetwork#7586]
### What we should change and why (this is tech debt)
Warning in rails 6.0 that will break in rails 6.1:`
DEPRECATION WARNING: `Module#parent` has been renamed to `module_parent`. `parent` is deprecated and will be removed in Rails 6.1. (called from _app_views_shared__footer_html_haml___1281714216389552113_47361922997640 at /home/runner/work/openfoodnetwork/openfoodnetwork/app/views/shared/_footer.html.haml:6)` 

### Context
rails 6 upgrade

### Impact and timeline
This will need to be fixed before the upgrade to rails 6.-1

**Comment by luisramos0** [comment]
I dont think this is actually breaking rails 6.1. Maybe it's not mandatory for the rails 6.1 upgrade.

**Comment by RachL** [comment]
@openfoodfoundation/core-devs do we still need this issue?
